internal/hierarchyconfig/validator_test.go

```diff
@@ -17,6 +17,71 @@ import (
 	"sigs.k8s.io/hierarchical-namespaces/internal/objects"
 )
 
+func TestManagedMeta(t *testing.T) {
+	f := foresttest.Create("-") // a
+	h := &Validator{Forest: f}
+	l := zap.New()
+	// For this test we accept any label or annotation not starting with 'h',
+	// to allow almost any meta - except the hnc.x-k8s.io labels/annotations,
+	// which cannot be managed anyway. And allows us to use that for testing.
+	if err := config.SetManagedMeta([]string{"[^h].*"}, []string{"[^h].*"}); err != nil {
+		panic(err)
+	}
```

Comment:
Why `panic`? I think `t.Fatal(err)` is probably better?

Reply:
Thanks for learning me how to do this! We use mostly Gomega/Ginkgo, and I have only done Golang for a year or so....